eva-accession-pipeline/src/main/java/uk/ac/ebi/eva/accession/pipeline/steps/tasklets/reportCheck/ReportCheckTasklet.java

```diff
@@ -136,9 +142,23 @@ private Variant readVcfIgnoringNonVariants(BufferHelper bufferHelper) throws Exc
                 }
             }
         }
+        if (variant != null) {
+            String contig = getEquivalentGenbankContig(variant);
+            variant = new Variant(contig, variant.getStart(), variant.getEnd(), variant.getReference(),
+                                  variant.getAlternate());
+        }
         return variant;
     }
 
+    private String getEquivalentGenbankContig(Variant variant) {
+        ContigSynonyms contigSynonyms = contigMapping.getContigSynonyms(variant.getChromosome());
+        if (ContigToGenbankReplacerProcessor.isReplacementPossible(variant, contigSynonyms, new StringBuilder())) {
```

Comment:
I did it in this way because in both eva-accession-import and eva-accession-pipeline we do contig replacements, and they are not exactly the same. The biggest difference is that the import takes into account whether the chromosome and chromosome_start are available. 

On the other hand, even though the import knows that the replacement is from refseq to genbank (or chr to genbank), and the accession pipeline is generic, I think this part can be refactored because both replace to genbank always, and I think the accession-pipeline.isReplacementPossible could use another conceptual review.

So I'll go ahead with the refactor